config/environments/development.rb

```diff
@@ -79,34 +95,37 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
-
-  # Use an evented file watcher to asynchronously detect changes in source code,
-  # routes, locales, etc. This feature depends on the listen gem.
-  if ENV['ENABLE_EVENT_FS_CHECKER'] == '1'
-    config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-  end
+  # config.i18n.raise_on_missing_translations = true
 
   # Devise requires a default URL
-  config.action_mailer.default_url_options = { host: ENV['HOSTNAME'], port: ENV['PORT'], protocol: :http }
+  config.action_mailer.default_url_options = { host: ENV['FQDN'], port: ENV['PORT'] }
 
   # don't need email sandbox with letter opener
   config.sandbox_email_mode = true
 
-  config.force_ssl = false
-  config.cache_store = :redis_cache_store, Rails.application.config_for(:cache_store).merge(expires_in: 8.hours)
-
-  # Web console from outside of docker
-  config.web_console.whitelisted_ips = ['172.16.0.0/12', '192.168.0.0/16', '10.0.0.0/8']
-
   # do gzip compressing in dev mode to simulate nginx config in production
   config.middleware.insert_after ActionDispatch::Static, Rack::Deflater
 
-  # Disable hostname checking for puma-dev
-  Rails.application.configure do
-    config.hosts.clear
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
+  if ENV['ENABLE_EVENT_FS_CHECKER'] == '1'
+    config.file_watcher = ActiveSupport::EventedFileUpdateChecker
   end
 
+  # Web console from outside of docker
+  config.web_console.allowed_ips = ['172.16.0.0/12', '192.168.0.0/16', '10.0.0.0/8']
+
+  console do
+    if ENV['CONSOLE'] == 'pry'
+      require 'pry-rails'
+      config.console = Pry
+    else
+      require 'irb'
+      config.console = IRB
+    end
+  end
 
-  I18n.config.available_locales = :en
+  # In order to fix the problem, the following options must be set.
```

Comment:
What problem is this fixing?

Reply:
I sure wish I knew.  That was a copy paste from the warehouse code.  The comment over there is something about databases, seems unrelated, also 3 years old.